sdk/lib/drivers/wdf/wdfldr/class.c

```diff
@@ -0,0 +1,822 @@
+/*
+ * PROJECT:     ReactOS WdfLdr driver
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     WdfLdr driver - class functions
+ * COPYRIGHT:   Copyright 2019 Max Korostil (mrmks04@yandex.ru)
+ */
+
+
+#include "class.h"
+#include "globals.h"
+
+#include <ntstrsafe.h>
+
+BOOLEAN
+NTAPI
+LibraryAcquireClientLock(
```

Comment:
Shouldn't Library* functions go to `library.c`?

Reply:
You right